docs/test-config.md

```diff
@@ -52,6 +52,8 @@ configurations:
       run_time: str # (1)!
       # * Test target URL 
       target_host: str
+      # * Test stop timeout
+      stop_timeout: str
```

Comment:
I have few points regarding this: 

First, the option appears in the code snippet, however, it doesn't have previous reference in the section explaining what are `vanilla_specs` (i.e. line 34 through 39). I would propose adding it and mention that it is optional next to it. Example 

```
vanilla_specs: object is a mandatory ...
...
- Task timeout [Optional field]
```

Second, I know that on _locust_ side, it is called "stop timeout", however, if we think about it from this tool user perspective, "stop timeout" is a bit miss leading. My proposal is to rename it (along with its references everywhere) to something like "task_timeout"

Third, in the doc, this is mentioned as str but in the code it is referenced and used as int. I don't have specific preference but it needs to be consistent. I think `int` make more sense overall.

Forth, the comment "# * Test stop timeout" I would strongly suggest changing it to something that would communicate more about the functionality of what it does. 

The main point here is consistency and user experience.

Reply:
Fixed